vm/actor_interface/src/builtin/account/mod.rs

```diff
@@ -42,6 +43,16 @@ pub fn is_v9_account_cid(cid: &Cid) -> bool {
     known_cids.contains(cid)
 }
 
+pub fn is_v10_account_cid(cid: &Cid) -> bool {
+    let known_cids = vec![
```

Comment:
Better to make it a static HashSet

Reply:
And even better to load these CIDs from a json/toml/yaml file

Reply:
It wouldn't be too good to load from file, since they don't change.